doc/release-notes/rl-2511.section.md

```diff
@@ -49,6 +49,8 @@
 
 - New hardening flags, `strictflexarrays1` and `strictflexarrays3` were made available, corresponding to the gcc/clang options `-fstrict-flex-arrays=1` and `-fstrict-flex-arrays=3` respectively.
 
+- `services.monero` now includes the `environmentFile` option for adding secrets to the Monero daemon config.
```

Comment:
Wrong place: this file is for Nixpkgs (apparently, I didn't even know there was one).
The one for NixOS is under `nixos/doc/manual/release-notes/`